src/test/javascript/spec/component/course/course-group.component.spec.ts

```diff
@@ -207,47 +215,50 @@ describe('Course Management Detail Component', () => {
             comp.courseGroup = CourseGroup.STUDENTS;
             comp.course = { ...course };
             comp.course.studentGroupName = 'testStudentGroupName';
-            expect(comp.courseGroupName).to.equal(comp.course.studentGroupName);
+            expect(comp.courseGroupName).toEqual(comp.course.studentGroupName);
```

Comment:
```suggestion
            expect(comp.courseGroupName).toBe(comp.course.studentGroupName);
```
Strings can be compared with toBe()
Please search yourself for other occurrences.

Reply:
I changed this in all occurrences.